Dockerfile

```diff
@@ -28,5 +28,7 @@ RUN apt-get -qqy update && apt-get install -qqy \
         kubectl && \
     gcloud config set core/disable_usage_reporting true && \
     gcloud config set component_manager/disable_update_check true && \
-    gcloud config set metrics/environment github_docker_image
+    gcloud config set metrics/environment github_docker_image && \
+    gcloud --version && \
```

Comment:
Let's separate this into another block:

```
# test tools
RUN gcloud --version && \
     ...
```

Reply:
That adds an additional layer, which I was trying to avoid.

Reply:
I'm aware, but that layer would be have no size/diff. Therefore it wouldn't be a big problem. But I guess this works too:

```
RUN foo && \
    # comment \
    bar
```